medical/models/medical_patient.py

```diff
@@ -95,6 +95,18 @@ def action_invalidate(self):
             ('z', 'law marriage'),
         ])
 
+    @api.multi
+    @api.onchange('state_id')
+    def onchange_state(self):
+        """ Hack to allow for onchange_state on partner.
+        Uses current user's partner, as there should be no correlation
+        between the partner and the response from this method anyways
+        """
+        res = self.env.user.partner_id.onchange_state(self.state_id.id)
```

Comment:
Ahhh this makes sense and totally clears up some issues I was running into when trying to figure out these onchanges for some of the other modules. Thanks for the explanation!

Seems our changes killed one of the tests, I'll rock it out in the morning.